modules/changing-the-log-level.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * cli/debugging-issues.adoc
+:_content-type: PROCEDURE
+[id="changing-the-log-level_{context}"]
+= Changing the log level
+
+.Procedure
+
+* Run the following command to change the log level:
++
+[source,terminal,subs="+quotes"]
+----
+$ roxctl "$ROX_CENTRAL_ADDRESS" central debug log --level=_<log_level>_ <1>
```

Comment:
The correct variable is `ROX_ENDPOINT` and we do not need to set this explicitly here.

Since we initially documented it together with the `ROX_API_TOKEN` we can drop this here.

If we want to keep it, let's use it like:
```bash
export ROX_ENDPOINT=<host:port>
roxctl image check --image=<image name>
```